be/src/Infrastructure/Catalog/Foundation.Catalog/Repositories/ProductIndexingRepositories/ProductIndexingRepository.cs

```diff
@@ -251,7 +255,12 @@ public async Task IndexAsync(Guid productId)
                     }
                 }
 
-                await this.elasticClient.BulkAsync(descriptor);
+                var response = await this.elasticClient.BulkAsync(descriptor);
+
+                if (response.IsValid is false)
+                {
+                    this.logger.LogDebug("Invalid response on indexing products in ProductIndexingRepository");
```

Comment:
**suggestion:** please use LogError and response.DebugInformation as a message, i.e. this.logger.LogError(response.DebugInformation);